pkg/proxy/proxy.go

```diff
@@ -325,17 +327,17 @@ func (p *Proxy) GetProxyPort(name string) (uint16, error) {
 	return 0, proxyNotFoundError(name)
 }
 
-// AllocateProxyPort() allocates a new port for listener 'name', or returns the current one if
+// AllocateCRDProxyPort() allocates a new port for listener 'name', or returns the current one if
 // already allocated.
 // Each call has to be paired with AckProxyPort(name) to update the datapath rules accordingly.
 // Each allocated port must be eventually freed with ReleaseProxyPort().
-func (p *Proxy) AllocateProxyPort(name string, ingress, localOnly bool) (uint16, error) {
+func (p *Proxy) AllocateCRDProxyPort(name string, localOnly bool) (uint16, error) {
```

Comment:
I wanted to rename this due to it creating a proxy port specific to CRD, to avoid the mistake of using this for any other purpose without further thought.